### ISSUE TITLE
problems in accuracy

### ISSUE THREAD

**Issue by loweite** [HWANG7308/R2Plus1D#1]
why i use the ucf101 split_1 rgb  with the pretrained model(r2.5d_d34_i32_ft_on_sports1m+kinetics.pth) the test accuracy is 85%   ,but which is 96% in paper .
**here is my setting:**
import argparse
parser = argparse.ArgumentParser(description="PyTorch implementation of Temporal Segment Networks")
parser.add_argument('--dataset',default='ucf101', type=str)#, choices=['ucf101', 'hmdb51', 'kinetics', 'cpipc'])
parser.add_argument('--modality', default="RGB", type=str, choices=['RGB', 'Flow', 'RGBDiff'])
parser.add_argument('--train_list',default="data/ucf101_rgb_train_split_2.txt", type=str)
parser.add_argument('--val_list',default="data/ucf101_rgb_val_split_2.txt", type=str)

# ========================= Model Configs ==========================
parser.add_argument('--arch', type=str, default="R2Plus1D-34", choices=['ResNet2D-18', 'ResNet3D-18', 'ResNet3D-34', 'R2Plus1D-34'])
parser.add_argument('--num_segments', type=int, default=1)
parser.add_argument('--pretrained_path', type=str, default='pretrained_models/r2.5d_d34_i32_ft_on_sports1m+kinetics.pth')
# if set true, it will not change the last fc in original pretrained weights,
# otherwise it will reinitialized the weights in last fc
parser.add_argument('--consensus_type', type=str, default='topk',
                    choices=['avg', 'max', 'topk', 'identity', 'rnn', 'cnn'])
parser.add_argument('--k', type=int, default=7)
parser.add_argument('--clip_length', type=int, default=32)
parser.add_argument('--sampling_middle', default=False, action="store_true")
parser.add_argument('--dropout', '--do', default=None, type=float,
                    metavar='DO', help='dropout ratio (default: None)')
parser.add_argument('--loss_type', type=str, default="nll",
                    choices=['nll'])

# ========================= Learning Configs ==========================
parser.add_argument('--epochs', default=60, type=int, metavar='N',
                    help='number of total epochs to run')
parser.add_argument('--iter_size', default=8, type=int, metavar='N',
                    help='That iter_size is equivalent to caffe iter_size')
parser.add_argument('-b', '--batch-size', default=4, type=int,
                    metavar='N', help='mini-batch size (default: 256)')
parser.add_argument('--lr', '--learning-rate', default=0.001, type=float,
                    metavar='LR', help='initial learning rate')
parser.add_argument('--schedule', default=[30, 40,50], type=float, nargs="+",
                    metavar='LRSteps', help='epochs to decay learning rate by 10')
parser.add_argument('--momentum', default=0.9, type=float, metavar='M',
                    help='momentum')
parser.add_argument('--weight-decay', '--wd', default=5e-4, type=float,
                    metavar='W', help='weight decay (default: 5e-4)')
parser.add_argument('--clip-gradient', '--gd', default=20, type=float,
                    metavar='W', help='gradient norm clipping (default: disabled)')
parser.add_argument('--no_partialbn', '--npb', default=False, action="store_true")

# ========================= Monitor Configs ==========================
parser.add_argument('--print-freq', '-p', default=20, type=int,
                    metavar='N', help='print frequency (default: 10)')
parser.add_argument('--eval-freq', '-ef', default=5, type=int,
                    metavar='N', help='evaluation frequency (default: 5)')

# ========================= Runtime Configs ==========================
parser.add_argument('-j', '--workers', default=4, type=int, metavar='N',
                    help='number of data loading workers (default: 4)')
parser.add_argument('--resume', default='', type=str, metavar='PATH',
                    help='path to latest checkpoint (default: none)')
parser.add_argument('-e', '--evaluate', dest='evaluate', action='store_true',
                    help='evaluate model on validation set')
parser.add_argument('--snapshot_pref', type=str, default="")
parser.add_argument('--start-epoch', default=0, type=int, metavar='N',
                    help='manual epoch number (useful on restarts)')
parser.add_argument('--gpus', nargs='+', type=int, default=None)
parser.add_argument('--flow_prefix', default="", type=str)

parser.add_argument('-c', '--checkpoint', default='checkpoint', type=str, metavar='PATH',
                    help='path to save checkpoint (default: checkpoint)')

**Comment by loweite** [comment]
split2
